package/src/Platform/Platform.web.tsx

```diff
@@ -1,6 +1,7 @@
 import type { RefObject, CSSProperties } from "react";
 import React, { useLayoutEffect, useMemo, useRef } from "react";
 import type { LayoutChangeEvent, ViewComponent, ViewProps } from "react-native";
+import { AssetRegistry } from "react-native";
```

Comment:
We cannot import react native here in `Platform.web.tsx`. The goal of this file is to not depend on React Native.

Reply:
Ah yes yes ofc. I thought I omitted this change to my commit, didn't mean to push this. Reverted it in the last commit. 

=> I only wanted to change `Image source is a number` to `Asset source is a number` to be more inclusive as per the use cases in which it could be thrown.